supabase/pending/new_ops_catalog_1_initial_migration.sql

```diff
@@ -1,50 +1,14 @@
--- Run this after deploying the updated agent.
+-- This migration can be run with no user-facing impact, and is the minimum needed for ops catalogs
+-- specs to be published with flowctl's new "don't publish the spec if it hasn't changed" machinery.
+-- It is intended to be run prior to deploying the new ops-catalog. It does not include clearing out
+-- the existing stats table, so ops-catalog should not yet be configured to publish the
+-- materialization.
 
 begin;
 
-drop table ops_catalog_template;
-drop function internal.create_ops_publication(tenant_prefix catalog_tenant, ops_user_id uuid);
 alter table live_specs add md5 text generated always as (md5(trim(spec::text))) stored;
 alter table tenants add l1_stat_rollup integer not null default 0;
 
--- Clear out the existing catalog_stats table and materialization metadata tables. The new
--- ops-catalog materialization will create new metadata tables. The catalog_stats table should be
--- created anew by re-running its migration prior to the new ops-catalog being deployed since the
--- connector would create the table differently than we want.
-drop schema catalog_stat_partitions;
-drop table catalog_stats, flow_checkpoints_v1, flow_materializations_v2;
-
--- Create a publication that will delete all the existing per-tenant ops catalogs.
-do $$
-declare
-    ops_user_id uuid;
-    current_tenant tenants;
-    new_draft_id flowid := internal.id_generator();
-    publication_id flowid := internal.id_generator();
-begin
-
-    select id into strict ops_user_id from auth.users where email = 'support@estuary.dev';
-    insert into drafts (id, user_id, detail) values
-    (new_draft_id, ops_user_id, 're-publishing ops catalog');
-    insert into publications (id, user_id, draft_id) values
-    (publication_id, ops_user_id, new_draft_id);
-
-    for current_tenant in
-        select * from tenants
-    loop
-        insert into draft_specs (draft_id, catalog_name, spec_type, spec)
-        select new_draft_id, catalog_name, null, null
-        from live_specs
-        where catalog_name in (
-            'ops/' || current_tenant.tenant || 'catalog-stats-view',
-            'ops/' || current_tenant.tenant || 'tests/catalog-stats',
```

Comment:
we _could_ restore this but _only_ delete `tests/catalog-stats`. Up to you whether that's actually expedient.